flybywiresim/a32nx
# fix(nd): degree symbol fix and add distance decimal logic

## Summary of Changes

- [x] Separated degree symbol from waypointTrack variable
- [x] Aligned the value and symbol for waypointTrack
- [x] Created a new class for DegreeSymbol
- [x] Separate distance display logic (greater than and less than 20)
- [x] Adjusted values to smoothen transition for VORDME distance info on ND

## Screenshots (if necessary)


### BEFORE:


![image](https://user-images.githubusercontent.com/29005005/110185290-027dcf80-7e38-11eb-876a-8f512f22878a.png)


### AFTER:
![image](https://user-images.githubusercontent.com/29005005/110188248-1a0d8600-7e41-11eb-9596-465b9fb347ef.png)

![image](https://user-images.githubusercontent.com/29005005/110188297-2bef2900-7e41-11eb-9cec-703d44c8eeeb.png)

![image](https://user-images.githubusercontent.com/29005005/110188338-4d501500-7e41-11eb-8cc4-6247d097338b.png)


## References





![image](https://user-images.githubusercontent.com/29005005/110185275-f3971d00-7e37-11eb-8f53-3c6fe2c480b1.png)

https://youtu.be/TEi-WX08ov8?t=17


Discord username (if different from GitHub): sidnov#8337

## Testing instructions


1. Spawn on a runway/C&D
2. Add a few waypoints, fly towards them manually/AP
3. Observe the Waypoint Track values change properly, and are not misaligned
4. Observe distance decimal values appear only when Waypoint<20NM
5. Additionally, you can also check VOR DME distance displayed on the bottom of the ND, updated with the same logic/value.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
